linetools/isgm/abscomponent.py

```diff
@@ -594,6 +598,49 @@ def repr_alis(self, T_kin=1e4*u.K, bturb=0.*u.km/u.s,
         s += '\n'
         return s
 
+    def repr_joevp(self, specfile, flags=(1,1,1), b_default=10*u.km/u.s):
```

Comment:
We need to avoid using '1' as a flag, as this value is reserved to denote line parameters that are to be fixed.  The other reserved value is '0', which denotes parameters that may freely vary, i.e., not tied to any other parameter.